src/v2/view-builder/views/email/BaseAuthenticatorEmailView.js

```diff
@@ -0,0 +1,86 @@
+import { loc, View, createCallout, _ } from 'okta';
+import BaseForm from '../../internals/BaseForm';
+import email from '../shared/email';
+import polling from '../shared/polling';
+import BaseAuthenticatorView from '../../components/BaseAuthenticatorView';
+import { SHOW_RESEND_TIMEOUT } from '../../utils/Constants';
+
+const ResendView = View.extend(
```

Comment:
Do we have a ticket to extract this into its own component? We have a few places now to be refactored I believe.

Reply:
https://oktainc.atlassian.net/browse/OKTA-317315